doc/shared/nvectors/NVector_Operations.rst

```diff
@@ -204,7 +204,7 @@ operations below.
    .. math::
       z_i = a x_i + b y_i, \quad i=0,\ldots,n-1.
 
-   The output vector *z* can be the same as either of the input vectors (*x* or *y*).
+   The output vector *z* can be the same as the input vector *x* but not *y*.
```

Comment:
```suggestion
   The output vector *z* can be the same as the input vector *x* but not *y*.
   
   .. versionchanged:: x.y.z
   
      Prior versions of SUNDIAS assumed that the output vector *z* could be used as 
      **either** of the input vectors (*x* or *y*).
```

Reply:
I think a user could easily interpret this statement as a breaking change. I.e., they may think "oh, to update to sundials X.Y.Z I need to go update my code now to ensure that z is not the same as y". I suggest:

```
Prior versions of SUNDIALS assumed that the output vector *z* could be used as  **either** of the input vectors (*x* or *y*). 
While all of the `N_Vector` implementations included with SUNDIALS still allow the output vector *z* to be the same as the input vector *x* or *y*, the API no longer requires it. 
```

Reply:
I like @balos1's suggestion.